docs/contents/community/faq.md

```diff
@@ -3,7 +3,7 @@
 ## A Kubernetes distribution?
 
 !!! question "Question"
-    What is a Kubernetes distribution and why do you call EKS Distro one?
+    What is a Kubernetes distribution and why do you call EKS Distro?
```

Comment:
This sentence doesn't make grammatical sense to me anymore. Maybe
>Why do you call EKS Distro a Kubernetes distribution?

Reply:
Or
> ... and how does EKS Distro classify as one?

Reply:
Done